vagrantnode.go

```diff
@@ -113,15 +120,10 @@ func (n *VagrantNode) RunCommandWithOutput(cmd string) (string, error) {
 
 // RunCommandBackground runs a background command in a vagrant node.
 func (n *VagrantNode) RunCommandBackground(cmd string) error {
-	var (
-		s   *ssh.Session
-		err error
-	)
-
-	if s, err = n.client.NewSession(); err != nil {
+	_, s, err := n.GetClientAndSession()
+	if err != nil {
 		return err
 	}
-	defer s.Close()
```

Comment:
that would eliminate the process, which we don't want. Instead we leak a connection and wait for it to clean up itself.

I'm not sure there's a better solution.